doc/config.md

```diff
@@ -283,6 +283,20 @@ updated and how often aggregated statistics are written to the log
 
 Default: 60
 
+### prepared_statement_cache_size
```

Comment:
I did some very basic benchmarks with pgbench and the results look quite good. No perf regressions, and prepared statements giving a ~35% perf boost over the simple query protocol.

initialized with scale 1, the table shows the best TPS of 3 runs and the following command:
```
pgbench --select-only --client 1 --jobs=1 -T 120 -P 5 --protocol <simple/extended/prepared>
```

<table>
<tr>
 <td>
 <td>
v1.20.0
 <td>
PR with cache_size=0
 <td>
PR with cache_size=100
<tr>
 <td>
simple
 <td>
7500
 <td>
7499
 <td>
7495
<tr>
 <td>
extended
 <td>
6973
 <td>
6853
 <td>
6867
<tr>
 <td>
prepared
 <td>
N/A
 <td>
N/A
 <td>
10147
</table>

The only thing that had me a bit worried was the extended value being lower, after the patch. But after doing some more runs that didn't hit that value on v1.20.0 it seems to have been a lucky outlier.

Reply:
Looks very promising! I haven't yet heard back from our team that will try to performance test this, but they did have some vacations coming up.

Reply:
@dashorst any updates from your team on performance testing with the current version?